src/Compilers/CSharp/Desktop/CommandLine/CommandLineParser.cs

```diff
@@ -1355,6 +1361,27 @@ private IEnumerable<CommandLineAnalyzerReference> ParseAnalyzers(string arg, str
             }
         }
 
+        private IEnumerable<CommandLineAnalyzerDependency> ParseAnalyzerDependencies(string arg, string value, List<Diagnostic> diagnostics)
+        {
+            if (value == null)
+            {
+                AddDiagnostic(diagnostics, ErrorCode.ERR_SwitchNeedsString, MessageID.IDS_Text.Localize(), arg);
+                yield break;
+            }
+            else if (value.Length == 0)
+            {
+                AddDiagnostic(diagnostics, ErrorCode.ERR_NoFileSpec, arg);
+                yield break;
+            }
+
+            List<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path)).ToList();
+
+            foreach (string path in paths)
+            {
+                yield return new CommandLineAnalyzerDependency(path);
```

Comment:
Do we need to check to see if the path points to a valid file on disk or do we just allow this to fail (with exception) in case where file does not exist? I guess just check to make sure that what we do here is consistent with what we would do for /r and /a.